README.md

```diff
@@ -1,65 +1,75 @@
+---
+page_type: sample
+languages:
+- javascript
+products:
+- azure
+description: "This QuickStart shows how to store a secret in Key Vault and how to retrieve it using a Web app. This web app may be  run locally or in Azure."
+urlFragment: key-vault-node-quickstart
+---
 
 # Quickstart: Set and retrieve a secret from Azure Key Vault using a Node Web App 
 
-This QuickStart shows how to store a secret in Key Vault and how to retrieve it using a Web app. This web app may be  run locally or in Azure. The quickstart uses Node.js and Managed service identities (MSIs)
+This Quickstart shows how to store a secret in Key Vault and how to retrieve it using a Web app. This web app may be run locally or in Azure. The Quickstart uses Node.js and Managed Service Identities (MSIs)
```

Comment:
Change "Managed Service Identities" to "Azure Managed Identities" - it was rebranded

Reply:
Do a search and replace in this docs for that.

The acronym MSI is also no longer valid, so can be removed.